FA.md

```diff
@@ -0,0 +1,45 @@
+NIP-FA
+======
+
+Kind-scoped follows
+-------------------
+
+`draft` `optional`
+
+This NIP defines kind `967`, a kind-scoped follow event.
+
+```jsonc
+{
+  "kind": 967,
+  "tags": [
+    ["p", "<followed-pubkey>", 'relay-url'],
+    ["k", "<some-kind>"],
+    ["k", "<some-other-kind>"]
+  ]
+}
+```
+
+Multiple `p` tags and multiple `k` tags are allowed.
+
+The `k` tag(s) define the scope of the follows.
+
+### Unfollow action
+
+Unfollowing is done by deleting the follow event, copying over the `k` tags from the follow event.
+
+```jsonc
+{
+  "kind": 5,
+  "tags": [
+    ["e", "<follow-event-id>"],
+    ["k", "<some-kind>"],
+    ["k", "<some-other-kind>"]
+  ]
+}
+```
```

Comment:
why is it necessary to copy the "k" tags? isn't this just a normal delete?

Reply:
allows a client that didn't do the delete to know that it's local state should be updated